react-carousel/my-app/src/App.tsx

```diff
@@ -0,0 +1,70 @@
+import './App.css';
+import { useEffect, useState } from 'react';
+import { Buttons } from './Buttons';
+import { Carousel } from './Carousel';
+import { PrevNextButton } from './PrevNextbutton';
+
+const images = [
+  {
+    src: './fushiguro.webp',
+    alt: 'Megumi Fushiguro',
+  },
+  {
+    src: './inumaki.webp',
+    alt: 'Toge Inumaki',
+  },
+  {
+    src: './itadori.webp',
+    alt: 'Yuji Itadori',
+  },
+  {
+    src: './kugisaki.webp',
+    alt: 'Nobara Kugisaki',
+  },
+  {
+    src: './panda.webp',
+    alt: 'Panda',
+  },
+  {
+    src: './zen-in.webp',
+    alt: "Maki Zen'in",
+  },
+];
+
+function App() {
+  const [currentIndex, setCurrentIndex] = useState(0);
```

Comment:
By putting all this logic in App, you don't have a nice, drop-in carousel that you can use wherever you need it. I'd like it reworked to give you such a component. That not only gives you practice at building React in the most productive way, but there is a good chance you'll want a carousel in your final project, and reworking it would give you one.

I don't think it will take long, and you usually get exercises done quickly, so I think it's worth your time to do it.